Content.Server/Speech/EntitySystems/DwarfAccentSystem.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class DwarfAccentSystem : EntitySystem
+{
+    // TODO:
+    // these are pretty bad to have as static dicts in systems, ideally these all get moved to prototypes
+    // these can honestly stay unlocalized in prototypes? -- most of these word-replacers make zero sense to localize into other languages
+    // since they're so english-specific
+    // all of the 'word-replacers' should also probably respect capitalization when transformed, so all caps -> all caps
+    // and first letter capitalized -> first letter capitalized, at the very least
+
+    // these specifically mostly come from examples of specific scottish-english (not necessarily scots) verbiage
+    // https://en.wikipedia.org/wiki/Scotticism
+    // https://en.wikipedia.org/wiki/Scottish_English
+    // https://www.cs.stir.ac.uk/~kjt/general/scots.html
+    private static readonly Dictionary<string, string> DirectReplacements = new()
+    {
+        { "girl", "lassie" },
+        { "boy", "laddie" },
+        { "man", "lad" },
+        { "woman", "lass" },
+        { "do", "dae" },
+        { "don't", "dinnae" },
+        { "dont", "dinnae" },
+        { "i'm", "A'm" },
+        { "im", "am"},
```

Comment:
yes this is all unlocalized, but see the comment. i was already working on a more general system for this because !soap but am splitting that out into its own PR to not scope creep